libs/mock-block-dock/src/service-module-callbacks.ts

```diff
@@ -34,7 +34,13 @@ const callExternalApiMethod = async (params: {
 
 export const constructServiceModuleCallbacks = (params: {
   blockProtocolApiKey?: string;
-}): EmbedderServiceMessageCallbacks => {
+}): Omit<
+  EmbedderServiceMessageCallbacks,
+  "openaiCreateImage" | "openaiCompleteText"
+> & {
+  openaiCreateImage?: never;
+  openaiCompleteText?: never;
+} => {
```

Comment:
These callbacks will be added in a future PR, and this type will then error when they are and can be reverted